pkg/server/plugin/nodeattestor/aws/client.go

```diff
@@ -78,17 +82,37 @@ func (cc *clientsCache) getClient(region string) (Client, error) {
 		return nil, status.Error(codes.FailedPrecondition, "not configured")
 	}
 
-	client, err := cc.newClient(cc.config, region)
+	var asssumeRoleArn bytes.Buffer
+	if cc.config.AssumeRoleArnTemplate != "" {
+		tmpl, err := template.New("assume-role-arn").Parse(cc.config.AssumeRoleArnTemplate)
+		if err != nil {
+			return nil, err
```

Comment:
all errors returned by plugins must be grpc status, can you return:
```
status.Errorf(codes.Internal, "failed to create tempalte: %v", err)
```
Same comment for all returned errors

Reply:
fixed